docs/source/en/using-diffusers/custom_pipeline_overview.md

```diff
@@ -161,6 +169,170 @@ out_lpw
   </div>
 </div>
 
+## Example community pipelines
+
+Community pipelines are a really fun and creative way to extend the capabilities of the original pipeline with new and unique features. You can find all community pipelines in the [diffusers/examples/community](https://github.com/huggingface/diffusers/tree/main/examples/community) folder with inference and training examples for how to use them.
```

Comment:
can we mention `from_pipe` here? https://huggingface.co/docs/diffusers/main/en/using-diffusers/loading#reuse-a-pipeline 

it is particularly useful for community pipelines because a lot of them do not have their own pre-trained checkpoint and is more like a "feature" that can be used to enhance one of our main pipelines (SD, SDXL)